WNPRC_EHR/resources/schemas/dbscripts/postgresql/wnprc-22.009-22.010.sql

```diff
@@ -0,0 +1,12 @@
+
+-- OCONNOR fixes
+ALTER TABLE oconnor.inventory ADD COLUMN IF NOT EXISTS ModifiedBy USERID;
```

Comment:
Because there are no module dependency relationships between WNPRC_EHR and OConnor, when bootstrapping this script may run before the `oconnor` schema has been created. Or there may be no OConnor module deployed at all. I know these aren't concerns on their production DB, but it will likely cause problems on TeamCity (see current test failures in this FB) and dev machines.

We should add these four lines as a new script in the OConnor module instead.